flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricRegistry.java

```diff
@@ -99,10 +99,17 @@ public MetricRegistry(Configuration config) {
 				DelegatingConfiguration reporterConfig = new DelegatingConfiguration(config, ConfigConstants.METRICS_REPORTER_PREFIX + namedReporter + ".");
 				final String className = reporterConfig.getString(ConfigConstants.METRICS_REPORTER_CLASS_SUFFIX, null);
 				if (className == null) {
-					LOG.error("No reporter class set for reporter " + namedReporter + ". Metrics might not be exposed/reported.");
+					LOG.error("No reporter class set for reporter {}. Metrics might not be exposed/reported.",namedReporter);
 					continue;
 				}
 
+				String delimiterForReporter = reporterConfig.getString(ConfigConstants.METRICS_REPORTER_SCOPE_DELIMITER,defaultDelimiter);
+				if(delimiterForReporter.length()!=1){
+					LOG.warn("Failed to parse delimiter for reporter {}, using global delimiter.",namedReporter);
+					delimiterForReporter=".";
+				}
+				this.delimiters.add(delimiterForReporter.charAt(0));
+
```

Comment:
Please add the configured delimiter to the info log message when a reporter was instantiated. (near the end of the constructor)